app/models/user.rb

```diff
@@ -107,6 +107,10 @@ def birth_date
     Formatters::DateFormatter.format_date(birth_date)
   end
 
+  def address
+    mpi.profile&.address
```

Comment:
Can you call `mpi_profile` here? It's a little silly right now, but the goal is to have the `mpi profile` calls going through a single function so we can check for nil (and possibly do other things like log the call in the future, etc)